lib/wallet/api/http/Cardano/Wallet/Api/Http/Shelley/Server.hs

```diff
@@ -3013,6 +3028,90 @@ constructTransaction api argGenChange knownPools poolStatus apiWalletId body = d
             . Map.toList
             . foldr (uncurry (Map.insertWith (<>))) Map.empty
 
+encryptionNonce :: ByteString
+encryptionNonce = "encrypt-data"
```

Comment:
[Nonce](https://en.wikipedia.org/wiki/Cryptographic_nonce) means "number used once". Here I don't see how this `encryptionNonce` will not be reused.